locksmith/src/fulfillment/dispatcher.ts

```diff
@@ -163,16 +161,13 @@ export default class Dispatcher {
       config.purchaserCredentials,
       provider
     )
+
     await walletService.connect(provider, walletWithProvider)
 
     // get lock
     const lock = await walletService.getLockContract(lockAddress)
 
-    // make sure reccuring payments are supported
```

Comment:
This check happened earlier... no need to check again. (simulation would have failed)